### PR TITLE
Update CHANGELOG.md

### LINKED ISSUE #26: fix the Unreleased URL on CHANGELOG.md
The URL must be `https://github.com/souhail-5/zeed/compare/LAST_RELEASE_TAG...main` instead of `https://github.com/souhail-5/zeed/compare/main...develop`

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
**Description**
Hi [souhail-5](https://github.com/souhail-5),
just trying to fix #26, please check it out

**Areas of Concern**
[CHANGELOG.md](https://github.com/souhail-5/zeed/blob/main/CHANGELOG.md)

**TODO**
[Update CHANGELOG.md](https://github.com/souhail-5/zeed/commit/c3906fcfc8303d1949f7e356dcccde7bdb50a31f)

**Dependencies / Relations with other projects**
- None

### CODE REVIEW

**Comment by souhail-5** [comment]
Hi @viktoriussuwandi 😃 ,

Thank you for your contribution. The notation `LAST_RELEASE_TAG` is a placeholder and not an actual tag. The latest release tag is: `v1.3.0`.

Would you be interested in updating your pull request?

PS: for a more automated approach, I'm considering a change to our GitHub Actions workflow, but we can handle this in another issue.

**Comment by viktoriussuwandi** [comment]
> Hi @viktoriussuwandi 😃 ,
> 
> Thank you for your contribution. The notation `LAST_RELEASE_TAG` is a placeholder and not an actual tag. The latest release tag is: `v1.3.0`.
> 
> Would you be interested in updating your pull request?
> 
> PS: for a more automated approach, I'm considering a change to our GitHub Actions workflow, but we can handle this in another issue.

sure @souhail-5, `LAST_RELEASE_TAG` updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/souhail-5/zeed/compare/main...develop)
+## [Unreleased](https://github.com/souhail-5/zeed/compare/v1.3.0)
```
Comment (souhail-5) [inline CHANGELOG.md:7]:
```suggestion
## [Unreleased](https://github.com/souhail-5/zeed/compare/v1.3.0...main)
```

The "Unreleased" section compares the latest tag to the `main` branch 😉.

**Review by souhail-5 — CHANGES_REQUESTED** [review]

**Review by souhail-5 — APPROVED** [review]

**Comment by souhail-5** [comment]
Thank you for your contribution @viktoriussuwandi ✌️.

**Pull request merged**